pkg/replication/replication.go

```diff
@@ -803,33 +855,52 @@ type ReplQueueStats struct {
 }
 
 // Workers returns number of workers across all nodes
-func (q ReplQueueStats) Workers() int64 {
-	var workers int64
+func (q ReplQueueStats) Workers() (tot WorkerStat) {
 	for _, node := range q.Nodes {
-		workers += int64(node.ActiveWorkers)
+		tot.Avg += node.Workers.Avg
+		tot.Curr += node.Workers.Curr
+		if tot.Max < node.Workers.Max {
+			tot.Max = node.Workers.Max
+		}
+	}
+	tot.Avg /= float32(len(q.Nodes))
```

Comment:
check for situations like `q.Nodes` is zero @poornas to avoid potential crashes if any.